lib/device_api/android/device/samsung.rb

```diff
@@ -0,0 +1,18 @@
+module DeviceAPI
+  module Android
+    # Samsung specific device class
+    class Samsung < Device
+      def initialize(options = {})
+
+        super
+        packages = list_installed_packages
+        multi_window = 'com.sec.android.app.FlashBarService'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.